srcs/app/server/Server.cpp

```diff
@@ -14,6 +14,83 @@ Server::~Server() {
 	close(listen_sd_);
 }
 
+void	Server::AcceptNewConnection() {
```

Comment:
No entiendo por qué habeis movido algunos métodos, no de archivo, sino dentro del mismo archivo, de algunas líneas más abajo a arriba, sin cambiar una coma, pero bueno. Hace la diff más confusa.Ya está hecho y no es plan de cambiarlo otra vez.

Reply:
Sorry, we move them to be in the same order as they are in the header file.